docs/users-guide/09-multi-series-charting.md

```diff
@@ -0,0 +1,85 @@
+06-sharing-answers.md
+
+# Charts with multiple series
+
+
+Data in isolation is rarely all that useful. One of the best ways to add context and clarity when communicating with data is to show data side-by-side with other data. Here are just a few  examples of data that is better together than apart.
+
+- Your company’s revenue vs. its costs over time
+- Average order price this month and user signups for that month
+- New users per day vs. returning users per day.
+- Orders per day per from a few different product lines.
+
+## In Metabase there are two main ways to get data side-by-side.
+
+1. Combining two existing saved questions that share a common dimension (like time) on a dashboard
+
+e.g. Let me see revenue over time and cost over time together.
+
+
+2. Asking a question that involves multiple dimensions in the query builder (or in SQL if you’re fancy).
+
+e.g. The count of users by region over time.
+
+
+## Combining two existing saved questions
+If you already have two or more saved questions you’d like to compare, and they share a dimension, they can be combined on any dashboard. Here’s how:
+
+1. Add a question with a dimension like time or a category to a dashboard. In practice, these will usually be line charts or bar charts.
+
+2. While in edit mode on the dashboard, hovering on the card will show a “Edit data” button. Click this button to start adding series that you want to compare to the first series..
+
+[ edit data control image ]
+
+3. In the Edit Data modal you’ll see the original question, and on the right you’ll see a list of compatible questions you can choose from. Check the box next to any questions you’d like to see side-by-side with the original, and Metabase will add it to the same chart.
+
+[ edit data modal image ]
+
+The X and Y axis will automagically update if necessary and Metabase will create a legend using the existing card titles to help you understand which question maps to which series on the chart. Repeat this process as many times as you need.
+
+To remove a series either uncheck the box, or click the x next to the title in the legend above the chart.
+
+Once you have your chart looking how you’d like, hit done and your changes will be shown on the card in the dashboard. Depending on how dense your data is, at this point you might want to consider enlarging your chart to make sure the data is legible.
+
+[end result image]
+
+
+### A quick note about SQL based questions.
+You’ll see SQL-based questions in the list of questions you can combine as well, but since SQL-based questions don’t have the same metadata we use for questions built using the query builder, we can’t always guarantee beforehand that they will be able to be added to the question you’re looking at. You’ll see a little warning next to SQL based cards to indicate this.
```

Comment:
This description feels really technical and almost over explains the situation.  You are also intermixing the terms "card" and "question".

I might try something a little less explanatory like ... "Metabase has less information about SQL based questions, so we cannot guarantee if they can be added reliably.  You'll see a little warning sign next to SQL questions to indicate this and when you try adding them just be aware it may not work."